mmdet/apis/ote/apis/detection/ote_utils/result_set.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
I propose to take corresponding module from old sdk

Reply:
It is not detachable as is.

Reply:
Do you mean `NullResultSet`, it should remain in the old sdk.